### DISCUSSION TITLE
Can't Change Page Hierarchy

### DISCUSSION THREAD

**Discussion by TravsVoid** [GetPublii/Publii#1618] [Bug Report] [answered]
### Operating system

Windows 11

### Publii version

0.46.0 (build 16890)

### Issue type

Application

### Bug description

Not sure if I'm doing something incorrectly, but I can't seem to move pages or create subpages.

https://github.com/user-attachments/assets/74df6e49-0294-4c75-b1e5-e052b048d754

### Steps to reproduce

The only thing I've done is create two Block Editor pages.

**Comment by bobmitro** [comment, marked as answer]
Indeed, the bug occurred when you created a page with the Block and Markdown editors. Please download [build 16904](https://getpublii.com/download/); it will work now.
